api/Config/Routes.js

```diff
@@ -18,6 +23,16 @@ let routes = (app) => {
     app.put('/availableCarParks', NoMethodController)
 
     app.delete('/availableCarParks', NoMethodController)
+
+    app.get('/bookCarPark', GetBookingController.getAllBookings)
+
+    app.get('/bookCarPark/:id', GetBookingController.getOneBooking)
+
+    app.post('/bookCarPark', [body('email').trim().isEmail().normalizeEmail(), body('registration').trim()], BookCarParkController)
+
+    app.put('/bookCarPark', NoMethodController)
+
+    app.delete('/bookCarPark', NoMethodController)
```

Comment:
This is more of a change suggestion than a request, but I think making these routes /bookings instead of bookCarPark would be slightly more RESTful than what you currently have.